### PR TITLE
Fixed Receive UI jumpiness

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,74 +40,74 @@
     }
   },
   "dependencies": {
-    "@babel/preset-env": "7.4.4",
-    "@react-native-community/async-storage": "1.3.3",
-    "@react-native-community/slider": "1.0.4",
+    "@babel/preset-env": "7.5.0",
```
Comment (Overtorment) [inline package.json]: Why did u up all deps?

File: package.json [hunk 2]
```diff
@@ -40,74 +40,74 @@
...
+    "prettier": "1.18.2",
     "process": "0.11.10",
     "prop-types": "15.7.2",
     "react": "16.8.6",
     "react-localization": "1.0.13",
     "react-native": "0.59.6",
-    "react-native-camera": "2.6.0",
-    "react-native-device-info": "1.6.0",
+    "react-native-camera": "2.11.0",
+    "react-native-device-info": "^2.2.2",
```
Comment (Overtorment) [inline package.json]: should be pinned

File: package.json (hunk 1 above)
Comment (Overtorment) [reply package.json]: Changelog of which upped deps have you read?
Even more, have you read diffs between versions? In our field of work I'm afraid we will have to, wallets will be a target.
So if you dont want to review lots of code of 3rd party deps lets make a rule to update onlye deps that need update

File: package.json [hunk 3]
```diff
@@ -40,74 +40,74 @@
...
-    "react-native-tcp": "3.3.0",
+    "react-native-svg": "9.5.1",
+    "react-native-tcp": "3.3.1",
     "react-native-tooltip": "git+https://github.com/marcosrdz/react-native-tooltip.git",
-    "react-native-vector-icons": "6.4.2",
+    "react-native-vector-icons": "6.6.0",
     "react-native-watch-connectivity": "0.2.0",
-    "react-native-webview": "5.8.1",
-    "react-navigation": "3.9.1",
+    "react-native-webview": "5.12.1",
```
Comment (Overtorment) [inline package.json]: Have you tested it? This one has a history of being non backward-compatible

File: screen/receive/details.js [hunk 4]
```diff
@@ -83,8 +83,8 @@ export default class ReceiveDetails extends Component {
   render() {
     return (
       <SafeBlueArea style={{ flex: 1 }}>
-        <View style={{}}>
-          <View style={{ marginTop: 32, justifyContent: 'center', alignItems: 'center', paddingHorizontal: 16 }}>
+        <View style={{ flex: 1, justifyContent: 'space-between' }}>
```
Comment (Overtorment) [inline screen/receive/details.js]: ok

File: ios/BlueWallet.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -829,10 +829,8 @@
 			buildActionMask = 2147483647;
 			files = (
 			);
-			inputFileListPaths = (
-			);
 			inputPaths = (
-				"${SRCROOT}/Pods/Target Support Files/Pods-BlueWallet/Pods-BlueWallet-resources.sh",
+				"${PODS_ROOT}/Target Support Files/Pods-BlueWallet/Pods-BlueWallet-resources.sh",
```
Comment (Overtorment) [inline ios/BlueWallet.xcodeproj/project.pbxproj]: is that ok that lots of `ios/` files changed? we didnt even update RN

Comment (marcosrdz) [reply ios/BlueWallet.xcodeproj/project.pbxproj]: its all Pods related. Basically syncing back to package.json for checksum.

File: package.json (hunk 3 above)
Comment (marcosrdz) [reply package.json]: Works.

File: package.json (hunk 1 above)
Comment (marcosrdz) [reply package.json]: Packages catching up to latest RN (pre-0.60).

**Pull request merged**